### PR TITLE
Add other risk question to risk

### PR DESCRIPTION
**Pull request by cesidio**
This PR will add other risk section to risks

[Trello](https://trello.com/c/0Z5MZmGg/135-3-new-question-other-risk-on-this-journey)

### CODE REVIEW

File: config/assessments_schema.yml [hunk 1]
```diff
@@ -1105,3 +1105,22 @@ assessment:
                 value: 'no'
               -
                 value: 'unknown'
+      other_risk:
+        questions:
+          -
+            name: other_risk
+            type: string
```
Comment (lostie) [inline config/assessments_schema.yml:1116]: This is missing the `Assessment::InclusionValidator` to ensure only the available answers are provided.

Comment (cesidio) [reply config/assessments_schema.yml:1116]: Thanks, fixed now

**Review by lostie — APPROVED** [review]
LGTM 👍

**Review by lostie — APPROVED** [review]
LGTM

**Pull request merged**